src/api/routes/transcripts.ts

```diff
@@ -122,10 +119,17 @@ export async function getSummariesAndNameMap(transcriptId: string): Promise<{
     }],
   });
 
-  for (const nm of snm) {
-    invariant(nm.user.name);
-    nameMap[nm.handlebarName] = `**${nm.user.name}**`;
+  let nameMap: TranscriptNameMap = tnm;
```

Comment:
This is the wrong way to do it. line 129 is changing `tnm` directly which is inappropriate. Use this instead:

```
const nameMap: TranscriptNameMap = [];

for (const handlebar of [...handlebarsSet]) {
  const entries = nameMap.filter(e => e.handlebarName === handlebar);
  nameMap.push(entries.length ? ... : ...);
}
```